src/browser/modules/Sidebar/ProjectsFilesScripts.tsx

```diff
@@ -169,7 +169,7 @@ function ProjectFilesScripts(props: ProjectFilesScripts): JSX.Element {
     scripts: projectFiles,
     isProjectFiles: true,
     scriptsNamespace: DOT,
-    title: 'Project Files',
+    title: 'Cypher files',
```

Comment:
just need to double check what we want to name this here (I cant recall if its optional), or the drawer header which will say `Project Files`.

Reply:
I tried removing it, and "Saved Scripts" is the default text it gets when I leave the title out

Reply:
Just need to get guidance on what we want to call stuff then really. Otherwise fine.